website/events/services.py

```diff
@@ -191,8 +191,9 @@ def update_registration(
         name = registration.name
 
     if (
-        not event_permissions(member, event, name)["update_registration"]
-        or not field_values
+        (not event_permissions(member, event, name)["update_registration"]
+        or not field_values)
+        and is_organiser(member, event)
```

Comment:
This if statement will continue if `field_values` is `None` and `member` isn't an organizer causing some tests to fail